spec/dummy/spec/packs_generator_spec.rb

```diff
@@ -4,6 +4,8 @@
 
 # rubocop:disable Metrics/ModuleLength
 module ReactOnRails
+  GENERATED_PACKS_OUTPUT_REGEX = /Generated Packs:/.freeze
```

Comment:
Why a regex? Seems like a simple string match.

Reply:
Updated to String